corehq/pillows/case.py

```diff
@@ -72,12 +72,25 @@ def transform_case_for_elasticsearch(doc_dict):
     return doc_ret
 
 
+def transform_case_transactions_to_actions(transactions):
+    def _action(trans):
+        return {
+            'server_date': trans['server_date'],
+            'date': trans['server_date'],
+            'xform_id': trans['form_id'],
+            'sync_log_id': trans['sync_log_id'],
+        }
+    return [_action(trans) for trans in transactions if not trans['revoked']]
```

Comment:
is there any reason the serializer can't just output the same stuff as the action? though i do like the names better in the transaction json